src/bitset.cpp

```diff
@@ -27,6 +27,51 @@ Bitset::Bitset(std::string str) : Bitset(str.length()) {
   }
 }
 
+// Note: Constructor for Subsplits.
+Bitset::Bitset(std::string str1, std::string str2)
```

Comment:
I'm not sure what this function does, but is it clear that we couldn't express this more simply in terms of other functions? Or algorithms?

Reply:
This was mostly for the proposed Subsplit class.  And I felt it made things more legible when writing test cases.

Reply:
Ah, cool!

@ognian- had a really nice idea about treating these as "views" of an underlying bitset object. I can't remember if there was any committed code for it though.

Reply:
Not questioning the need for such a thing. Curious if we could implement it more cleanly.

Reply:
Moved to Bitset::Subsplit() and added clarifying lines.